src/styles/variables.css

```diff
@@ -0,0 +1,73 @@
+/* Media Queries */
+@custom-media --breakpoint-not-small screen and (min-width: 30em);
+@custom-media --breakpoint-medium screen and (min-width: 30em) and (max-width: 60em);
+@custom-media --breakpoint-large screen and (min-width: 60em);
+
+/* Colours */
+:root {
+  --black: #000;
```

Comment:
I don't have a strong opinion either way. Some people asked about customization so I put variables in there. I'm ok with taking it out.